trade-finance-manager-api/src/v1/validation/validateIds.js

```diff
@@ -13,8 +13,8 @@ const isValidRegex = (regex, value) => regex.test(value);
 /**
   Validates if a value is a valid mongo id
 
- * @param mongoId - the value to validate
- * @returns Boolean - true if valid, false if not
+ * @param {string} mongoId - the value to validate
```

Comment:
I think I would probably type this as `string || ObjectId`? Consider the function uses `mongoId.toString()` this might have initially been intended to validate the `ObjectId` version.